LayoutTests/fast/attachment/mac/attachment-element-gpu-process.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html><!-- webkit-test-runner [ AttachmentElementEnabled=true UseGPUProcessForDOMRenderingEnabled=true ] -->
```

Comment:
This test will stop testing anything meaningful when UseGPUProcessForDOMRenderingEnabled is turned on by default.

Reply:
I explicitly set `UseGPUProcessForDOMRenderingEnabled = false` in the expected html. This does beg the question of whether we still want to test the non gpup code path after this is enabled. Maybe it would make sense to remove this test once it is enabled by default.

Reply:
A better test would be to compare <attachment> rendering against some content that is not <attachment> but has similar rendering.